src/app/views/river-detail/components/flow-tab/flow-tab.vue

```diff
@@ -438,6 +448,12 @@ export default {
     },
     handleOpenGageModal () {
       this.$refs.gageLinkModal.open()
+    },
+    calendarView () {
+      this.releaseView = "calendar"
```

Comment:
eric consider enums or consts something like : 
`const FLOWVIEW_CALENDAR=1; const FLOWVIEW_TABLE=2`
or
`const FLOWVIEWENUM = Object.Freeze({CALENDAR:1, TABLE:2});`

then assign releaseView to FLOWVIEWENUM.CALENDAR rathern than the string. 

strings have too many gotchas and they don't constrain to the limits of the number of views you want to represent.